app/helpers.py

```diff
@@ -0,0 +1,68 @@
+from functools import wraps
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, make_response
+
+# Decorator to check if task exists.
+def require_valid_task_id(endpoint):
+    @wraps(endpoint)
+    def fn(*args, task_id, **kwargs):
+
+        task = Task.query.get(task_id)
+
+        if not task: 
+            return make_response("", 404)
```

Comment:
This actually returns an html response rather than JSON. The tests end up being ok with this since they look for `None` in the `get_json` call, and if the response _isn't_ JSON, that's what `get_json` returns. However, we should really return a JSON response (since we are building a JSON api), such as
```python
            return jsonify(None), 404
```
Ideally, we should be returning a more detailed response, but the tests are a little weird.